lib/spack/spack/build_systems/cuda.py

```diff
@@ -93,12 +93,16 @@ def cuda_flags(arch_list):
     conflicts('%clang@7:', when='+cuda ^cuda@10.0.130' + arch_platform)
 
     # Intel is mostly relevant for x86_64 Linux, even though it also
-    # exists for Mac OS X.
-    conflicts('%intel@:14,16:', when='+cuda ^cuda@7.5')
-    conflicts('%intel@:14,17:', when='+cuda ^cuda@8.0.44')
-    conflicts('%intel@:14,18:', when='+cuda ^cuda@8.0.61:9.1')
-    conflicts('%intel@17:18', when='+cuda ^cuda@9.2:')
-    conflicts('%intel@19:', when='+cuda')
+    # exists for Mac OS X. No information prior to CUDA 3.2 or Intel 11.1
+    conflicts('%intel@:11.0', when='+cuda ^cuda@:3.1')
+    conflicts('%intel@:12.0', when='+cuda ^cuda@5.5:')
+    conflicts('%intel@:13.0', when='+cuda ^cuda@6.0:')
+    conflicts('%intel@:13.2', when='+cuda ^cuda@6.5:')
+    conflicts('%intel@:14.9', when='+cuda ^cuda@7:')
+    # Intel 15.x is compatible with CUDA 7 thru current CUDA
+    conflicts('%intel@16.1:', when='+cuda ^cuda@:7.5.18')
+    conflicts('%intel@17.1:', when='+cuda ^cuda@:8.0.44')
```

Comment:
maybe as `%intel@:14.9,16.1:`? supported are 15.0.4-16.0 in CUDA 8.0.44 

maybe `%intel@:14.9,17.1:` as 15.0.5-17.0 are supported in CUDA 8.0.61 and 9.0 and 9.1 and 9.2

Furthermore, CUDA 10.0 also supports up to ICC 18.0 and CUDA 10.1 supports up to ICC 19.0